tests/foreman/cli/test_remoteexecution.py

```diff
@@ -197,3 +235,17 @@ def test_positive_run_scheduled_job_template(self):
 
         @Assert: Verify the job was successfully ran after the designated time
         """
+        system_current_time = ssh.command('date +"%b %d %Y %I:%M%p"').stdout[0]
```

Comment:
Let me know your decision here. You can try to use out standalone job on Jenkins for testing if you like to have similar env to our test env.